processing file asmFiles/test.coherence1.asm (1 of 6)
diff: memcpu.hex: No such file or directory
processing file asmFiles/test.coherence2.asm (2 of 6)
diff: memcpu.hex: No such file or directory
processing file asmFiles/test.msi1.asm (3 of 6)
diff: memcpu.hex: No such file or directory
processing file asmFiles/test.msi2.asm (4 of 6)
diff: memcpu.hex: No such file or directory
processing file asmFiles/test.msi3.asm (5 of 6)
diff: memcpu.hex: No such file or directory
processing file asmFiles/test.msi4.asm (6 of 6)
diff: memcpu.hex: No such file or directory
processing file asmFiles/test.coherence1.asm (1 of 6)
processing file asmFiles/test.coherence2.asm (2 of 6)
--- memsim.hex	2023-11-21 13:26:34.334822047 -0500
+++ memcpu.hex	2023-11-21 13:26:34.320821524 -0500
@@ -20,6 +20,5 @@
 :04008700FFFFFFFF79
 :0401000000000001FA
 :04010200DEADBEEFC1
-:040103000000DEAD6D
 :040105000000BEEF49
 :00000001FF
processing file asmFiles/test.msi1.asm (3 of 6)
processing file asmFiles/test.msi2.asm (4 of 6)
processing file asmFiles/test.msi3.asm (5 of 6)
processing file asmFiles/test.msi4.asm (6 of 6)
processing file asmFiles/test.coherence1.asm (1 of 6)
processing file asmFiles/test.coherence2.asm (2 of 6)
processing file asmFiles/test.msi1.asm (3 of 6)
processing file asmFiles/test.msi2.asm (4 of 6)
processing file asmFiles/test.msi3.asm (5 of 6)
processing file asmFiles/test.msi4.asm (6 of 6)
